### DISCUSSION TITLE
A new subtype of TVEpisode: TVStandalone

### DISCUSSION THREAD

**Discussion by gmackenz** [schemaorg/schemaorg#2939] [General]
Entity Type: TVStandalone

Intended purpose: Capture standalone/one-off shows that don't fit into the traditional TVSeries/TVEpisode/TVSeason model: breaking news, election specials, documentary specials, TVShows with just 1 episode such as Frosty The Snowman, any other standalone TV episode that doesn’t connect to any TVSeries as such.

This is a proposal from Google as a means to help identify and better handle such special television broadcast content.

**Comment by WeaverStever** [comment]
There is also an open issue on Play, the Type is empty and needs to be normalized to include properties like actor, director, production company, venue etc. Currently, Type play is empty and only includes properties form Thing and CreativeWork.

A play can be an Event, but it can also be a collection of events.

Normalize CreativeWork Play #2918

**Comment by gmboxsmg2psn-collab** [comment]
En OroBet Ecuador [https://orobet.ec/es/casino/cold-games](https://orobet.ec/es/casino/cold-games) , la sección de Cold Games permite explorar distintas propuestas de casino online desde una interfaz organizada. Es una opción para quienes quieren descubrir juegos con temáticas diferentes y revisar sus características antes de jugar, siempre de manera responsable.
